map_gui/src/load.rs

```diff
@@ -312,8 +312,10 @@ mod wasm_loader {
         }
     }
 
-    // TODO This is a horrible copy of FileLoader. Make the serde FileLoader just build on top of
-    // this one!!!
+    /// This loads a file without deserializing it.
+    ///
+    /// TODO This is a horrible copy of FileLoader. Make the serde FileLoader just build on top of
+    /// this one!!!
     pub struct RawFileLoader<A: AppLike> {
```

Comment:
I'm not sure that I tested this properly - do you have a suggestion for how to test the RawFileLoader?

Reply:
This is only used for fonts: https://github.com/a-b-street/abstreet/blob/3c601b50c19f7ee92a5ff4021edc053c2f0a1f6a/map_gui/src/load.rs#L44

Open http://d2w1n8k2hsp0xg.cloudfront.net/dev/abstreet.html?--dev&ly/tripoli/maps/center.bin, zoom in on some roads, and make sure you see some Arabic. That particular URL doesn't work because `system/ly` isn't in your S3 dir, but that's the idea.

(Rest of review tomorrow; I over-promised on the "tonight" part)

Reply:
Thanks for the tip!

I've copied that file over and I'm seeing arabic text rendered - does that mean it worked?
http://d2w1n8k2hsp0xg.cloudfront.net/dev/abstreet.html?--dev&ly/tripoli/maps/center.bin

<img width="618" alt="Screen Shot 2021-04-01 at 9 53 18 AM" src="https://user-images.githubusercontent.com/217057/113327756-31208600-92d0-11eb-96bf-2ff16854e9c4.png">

Reply:
Yup, all good! `RawFileLoader` is a different beast from the rest so that it can return raw `Vec<u8>` bytes and pass it to usvg. Looking fine.